mlflow/models/evaluation/base.py

```diff
@@ -533,13 +590,16 @@ def evaluate(
             "an instance of `mlflow.pyfunc.PyFuncModel`."
         )
 
+    evaluator_name_list, evaluator_name_to_conf_map = \
+        _normalize_evaluators_and_evaluator_config_args(evaluators, evaluator_config)
+
     with _start_run_or_reuse_active_run(run_id) as actual_run_id:
         client = mlflow.tracking.MlflowClient()
         dataset._log_dataset_tag(client, actual_run_id)
 
         eval_results = []
-        for evaluator_name in evaluators:
-            config = evaluator_config.get(evaluator_name) or {}
+        for evaluator_name in evaluator_name_list:
```

Comment:
If `evaluator_name_list` is empty, can we throw an exception with error code `INVALID_PARAMETER_VALUE` to tell the user that the model couldn't be evaluated by any of the registered evaluators? Can we also tell the user to check that their `model_type` is accurate? We currently fail silently, which was confusing when accidentally I passed in `model_type="regression"` instead of `model_type="regressor"` and got no output.